test/extensions/wasm/wasm_test.cc

```diff
@@ -25,7 +25,11 @@ class TestContext : public Extensions::Common::Wasm::Context {
 public:
   TestContext(Extensions::Common::Wasm::Wasm* wasm) : Extensions::Common::Wasm::Context(wasm) {}
   ~TestContext() override {}
-  MOCK_METHOD2(scriptLog, void(spdlog::level::level_enum level, absl::string_view message));
+  void scriptLog(spdlog::level::level_enum level, absl::string_view message) override {
+    std::cerr << std::string(message) << "\n";
```

Comment:
Left-over from debug?

Reply:
Sort of.  I generally find it useful.  Is extra output in tests discouraged in envoy?  If so I will remove it.  I can also convert it to an envoy_log(debug) if that is more standard.

Reply:
I asked, because I wasn't sure if that's intentional or not, but it looks like it's used in a few tests in Envoy, so feel free to leave it.